packages/governance-sdk/tests/governance/api.v4.test.ts

```diff
@@ -0,0 +1,78 @@
+import { Keypair } from '@solana/web3.js';
+import { GoverningTokenConfigAccountArgs, GoverningTokenType } from '../../src';
+import { BenchBuilder } from '../tools/builders';
+
+test('setRealmConfig', async () => {
```

Comment:
These tests should be in V2.V3 because they should work for spl-gov V2 and V3 and there is no V4 yet